policy/datasets.yaml

```diff
@@ -415,6 +415,20 @@ ref_cat:
     storage: FitsCatalogStorage
     table: ignored
     template: ref_cats/%(name)s/%(pixel_id)s.fits
+starGalaxy_classifier:
+    description: "The trained star galaxy separation classfier - uses colors and morphology information"
+    persistable: ignored
+    python: sklearn.calibration.CalibratedClassifierCV
+    storage: PickleStorage
+    table: ignored
+    template: algorithmData/starGalaxyClassifier/%(version)s/clfIsotonicColour.pickle
+starGalaxy_morphOnlyClassifier:
+    description: "The trained star galaxy separation classfier - uses morphology information only"
+    persistable: ignored
+    python: sklearn.calibration.CalibratedClassifierCV
+    storage: PickleStorage
+    table: ignored
+    template: algorithmData/starGalaxyClassifier/%(version)s/clfIsotonicMorph.pickle
```

Comment:
These two classifiers appear to be the only use of a `version` keyword in a file path template. I'm wondering if it's strictly necessary or if there is a better way to handle versioning than via file paths?

Reply:
Good catch.  It'd be worth talking to @TallJimbo about Gen3 butler approaches to this.

Reply:
FWIW, I'm okay with this approach, at least right now (I'd discussed it with @sr525 previously), and it's not obvious the answer will change in Gen3, though the current nomenclature would be to use "label" rather than "version" as the data ID key.  There will be at least two other possible alternative approaches in Gen3 (different collections and different DatasetTypes), but at least right now it feels like this will remain the best approach there.

Reply:
Fair enough, thanks for this context. I prefer `label`, so if we can please switch to that instead of `version`, I'll consider this reviewed.